aws/scripts/py/reports.py

```
@@ -106,48 +147,86 @@
 
 
 def write_to_file(data):
-    json_string = json.dumps(data)
-    open('payload.json', 'w').write(json_string)
+    try:
+        json_string = json.dumps(data)
+        with open('payload.json', 'w') as file:
+            file.write(json_string)
+    except (IOError, json.JSONEncodeError) as e:
```

Ruff

Replace aliased errors with `OSError`

Replace with builtin `OSError`

(UP024)